compiler/src/test/resources/input_file/bwa-mem-tool_input.json

```diff
@@ -24,6 +24,6 @@
   "bwa-mem-tool.minimum_seed_length": 3,
   "bwa-mem-tool.args.py": {
     "class": "File",
-    "location": "dx://file-G0G3BZQ0yzZf6x1Y3vxzpgk6"
+    "location": "dx://project-Fy9QqgQ0yzZbg9KXKP4Jz6Yq:file-G0G3BZQ0yzZf6x1Y3vxzpgk6"
```

Comment:
Do the tests pass without being qualified with the project (as before)? Or is this update only an optimization?